.travis.yml

```diff
@@ -5,39 +5,39 @@ sudo: required
-  - ES=1 TEST_GROUP="cucumber -r features features/other_a            -f Ao3Cucumber::Formatter --color"
-  - ES=1 TEST_GROUP="cucumber -r features features/other_b            -f Ao3Cucumber::Formatter --color"
-  - ES=1 TEST_GROUP="cucumber -r features features/prompt_memes_a     -f Ao3Cucumber::Formatter --color"
-  - ES=1 TEST_GROUP="cucumber -r features features/prompt_memes_b     -f Ao3Cucumber::Formatter --color"
-  - ES=1 TEST_GROUP="cucumber -r features features/prompt_memes_c     -f Ao3Cucumber::Formatter --color"
-  - ES=1 TEST_GROUP="cucumber -r features features/search             -f Ao3Cucumber::Formatter --color"
-  - ES=1 TEST_GROUP="cucumber -r features features/tag_sets           -f Ao3Cucumber::Formatter --color"
-  - ES=1 TEST_GROUP="cucumber -r features features/tags_and_wrangling -f Ao3Cucumber::Formatter --color"
-  - ES=1 TEST_GROUP="cucumber -r features features/users              -f Ao3Cucumber::Formatter --color"
-  - ES=1 TEST_GROUP="cucumber -r features features/works              -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="rspec spec"
-  # - ES=2 TEST_GROUP="cucumber -r features features/admins             -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/bookmarks          -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/collections        -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/comments_and_kudos -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/gift_exchanges     -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/importing          -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/other_a            -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/other_b            -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/prompt_memes_a     -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/prompt_memes_b     -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/prompt_memes_c     -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/search             -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/tag_sets           -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/tags_and_wrangling -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/users              -f Ao3Cucumber::Formatter --color"
-  # - ES=2 TEST_GROUP="cucumber -r features features/works              -f Ao3Cucumber::Formatter --color"
+  - ES=1 TEST_GROUP="cucumber --tag ~@new-search -r features features/admins             -f Ao3Cucumber::Formatter --color"
```

Comment:
The `~` character indicates negation. The ES=1 tests exclude any cucumber tests tagged `@new_search`, while the ES=2 tests exclude any cucumber tests tagged `@old_search`.